app/models/report.rb

```diff
@@ -301,6 +301,10 @@ def record_empty_theses(collection)
     }
   end
 
+  def list_expired_holds(collection)
+    collection.active_or_expired.ends_today_or_before.order(:date_end)
```

Comment:
Related suggestion to calling the scope in the controller, if we do that I think we could move this to:

`collection.ends_today_or_before.order(:date_end)`